payroll-processor-functions/.vscode/tasks.json

```diff
@@ -81,7 +81,7 @@
       "detail": "Runs the Payroll Processor function via Azure Function Core Tools on http://localhost:7071",
       "dependsOn": "Function: Build (Debug)",
       "options": {
-        "cwd": "${workspaceFolder}/src/payroll-processor-functions/bin/Debug/netcoreapp3.1/bin"
+        "cwd": "${workspaceFolder}/src/payroll-processor-functions/bin/Debug/netcoreapp3.1"
```

Comment:
👍

Reply:
i think this fix is in all the open PRs lol